--- build/src/MAKE/Makefile.serial
+++ build/src/MAKE/Makefile.serial
@@ -6,17 +6,17 @@ SHELL = /bin/sh
 # compiler/linker settings
 # specify flags and libraries needed for your compiler

-CC =		g++
-CCFLAGS =	-g -O3
+# CC =		g++
+CCFLAGS =	-g -O3 -fpermissive
 SHFLAGS =	-fPIC
 DEPFLAGS =	-M

-LINK =		g++
+# LINK =		g++
 LINKFLAGS =	-g -O
 LIB =
 SIZE =		size

-ARCHIVE =	ar
+# ARCHIVE =	ar
 ARFLAGS =	-rc
 SHLIBFLAGS =	-shared -rdynamic