--- build/src/lammps.cpp	2016-05-12 08:54:17.000000000 -0500
+++ build/src/lammps.cpp	2025-02-12 16:52:10.525996425 -0600
@@ -63,6 +63,9 @@ static void print_style(FILE *fp, const char *str, int &pos);
 
 LAMMPS::LAMMPS(int narg, char **arg, MPI_Comm communicator)
 {
+  for (int i = 0; i < narg; i++) {
+    printf("EARLY: ARGS: [%s]\n", arg[i]);
+  }
   memory = new Memory(this);
   error = new Error(this);
   universe = new Universe(this,communicator);
@@ -104,6 +107,7 @@ LAMMPS::LAMMPS(int narg, char **arg, MPI_Comm communicator)
 
   int iarg = 1;
   while (iarg < narg) {
+    fprintf(universe->uscreen, "INFO[AAYUSH]: [%s]\n", arg[iarg]);
     if (strcmp(arg[iarg],"-partition") == 0 ||
         strcmp(arg[iarg],"-p") == 0) {
       universe->existflag = 1;
