--- build/src/comm.cpp
+++ build/src/comm.cpp
@@ -85,7 +85,7 @@ Comm::Comm(LAMMPS *lmp) : Pointers(lmp)
   if (lmp->kokkos) {
     nthreads = lmp->kokkos->num_threads * lmp->kokkos->numa;
   } else if (getenv("OMP_NUM_THREADS") == NULL) {
-    nthreads = 1;
+    nthreads = omp_get_max_threads();
     if (me == 0)
       error->message(FLERR,"OMP_NUM_THREADS environment is not set. "
                            "Defaulting to 1 thread.");
